pkg/lib/cockpit.js

```diff
@@ -186,7 +186,7 @@ function Channel(options) {
 
     self.wait = function wait(callback) {
         if (!waiting) {
-            waiting = cockpit.defer();
+            waiting = new Deferred();
```

Comment:
I just had a déjà vu, and indeed two commits earlier "cockpit.js: avoid a cockpit.defer() call" did the same thing. Let's squash these two.

Reply:
Sure.

Reply:
Done.